docs/source/cli_help/status.rst

```diff
@@ -82,4 +82,26 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 
  .. note::
 
-    You can add additional space after ``└─`` and before branch name by setting git config key ``machete.status.extraSpaceBeforeBranchName``.
+    To make it easier to select branch name from the ``status's`` output on certain terminals (E.g. Alacritty), you can add additional
+    space between ``└─`` and ``branch name`` by setting git config key ``machete.status.extraSpaceBeforeBranchName`` = ``true``.
+
+    E.g.
+        default:
+
+        .. code-block:: git
```

Comment:
Hmm is there `git` language supported by Sphinx/ReadTheDocs/whatever renders the syntax rendering? 🤔 
How does it even differ from vanilla "un-languaged" `.. code-block::`?

Reply:
Not much, the font is just a bit darker

Reply:
Well, to answer the question about support afaics sphinx uses https://pygments.org/docs/lexers/ and i cant find `git` there but it looks like it works so i guess it's supported otherwise it would have thrown an error as it did for `gitconfig`:
`docs/source/github_config_keys.rst:19: WARNING: Pygments lexer name 'gitconfig' is not known`

Reply:
I wrote a test gist with this code and it renders same way as `none` lexer.

I can't find a list of supported highlights for RST on GitHub (for markdown it's [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)). I don't see `git` neither as supported [on Pygments page](https://pygments.org/languages/) nor as supported for GitHub Flavoured Markdown.

Reply:
> the font is just a bit darker

Okay... so if it significantly improves readability, then it can stay... otherwise I'd remove coz it's surprising/unexpected ;)

Reply:
Yes, IMO it improves readability so i think it should stay